config/src/main/java/org/hyperledger/besu/config/NetworkDefinition.java

```diff
@@ -38,7 +38,7 @@ public enum NetworkDefinition {
    * href="https://docs.linea.build/get-started/how-to/run-a-node/besu">Linea Besu developer
    * info</a>
    */
-  LINEA("/linea-mainnet.json", BigInteger.valueOf(59144), true, true),
+  LINEA_MAINNET("/linea-mainnet.json", BigInteger.valueOf(59144), true, true),
```

Comment:
this is effectively a breaking change since this is already released for linea, right? 
Would we want to support `besu --network=linea` as well as `besu --network=linea-mainnet`

Reply:
thanks for confirming. approved.